Include/internal/pycore_magic_number.h

```diff
@@ -287,6 +287,7 @@ Known values:
     Python 3.15a1 3654 (Fix missing exception handlers in logical expression)
     Python 3.15a1 3655 (Fix miscompilation of some module-level annotations)
     Python 3.15a1 3656 (Add TRACE_RECORD instruction, for platforms with switch based interpreter)
+    Python 3.15a1 3657 Lazy imports IMPORT_NAME opcode changes
```

Comment:
```suggestion
    Python 3.15a3 3657 Lazy imports IMPORT_NAME opcode changes
```

I _think_ we already released 3.15a2 right?